src/schemas/sassSchemaUnits.ts

```diff
@@ -0,0 +1,127 @@
+const sassSchemaUnits = [
+  {
+    name: 's',
+    body: '$1s',
```

Comment:
I don't feel it's necessary to have the tab stop return to the value,  especially since the snippet includes the value already. I would expect to be able to hit return or tab right after the snippet had been selected and this currently just separates the value from the unit.